src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/OrderReturnStates/OrderReturnStateType.php

```diff
@@ -41,13 +41,22 @@
  */
 class OrderReturnStateType extends TranslatorAwareType
 {
+    protected const NAME_CHARS = '!<>,;?=+()@#"{}_$%:';
+
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('name', TranslatableType::class, [
+                'label' => $this->trans('Status name', 'Admin.Shopparameters.Feature'),
```

Comment:
Not sure if typo, or used like that. (applies to all usages in this file)

```suggestion
                'label' => $this->trans('Status name', 'Admin.ShopParameters.Feature'),
```

Reply:
It's not a typo. Check in `MetaController` or `AdminStatusesController`.

Reply:
@FabienPapet Not everything is came case! :D